ProjectShop/ProductApp/tests/test_models.py

```diff
@@ -0,0 +1,182 @@
+from django.test import TestCase
+from django.db.utils import IntegrityError
+
+from ProductApp.tests.factories import *
+
+
+class ProductTestCase(TestCase):
+	"""Test Case for Product model"""
+	def setUp(self) -> None:
+		pass
```

Comment:
What does the blank SetUp method do? Sofia has written literally the same method. I wonder who's copying who.